### DISCUSSION TITLE
Did Corona have an impact on the number of PyScaffold downloads?

### DISCUSSION THREAD

**Discussion by FlorianWilhelm** [pyscaffold/pyscaffold#437] [Show and tell]
This is not meant to be taken overly serious ;-) My main motivation was to play a little bit around with [BigQuery](https://console.cloud.google.com/bigquery) and [Colab](https://colab.research.google.com/). Getting the number of monthly downloads of any PyPI package is as easy as:
```sql
SELECT 
  COUNT(*) AS num_downloads,
  DATE_TRUNC(DATE(timestamp), MONTH) AS `month`,
FROM `bigquery-public-data.pypi.file_downloads`
WHERE file.project = 'pyscaffold'
    AND details.installer.name = 'pip'
    AND DATE(timestamp) BETWEEN DATE('2016-01-01') AND CURRENT_DATE()
GROUP BY `month`
ORDER BY `month`
```
Saving this as CSV and applying the usual Pandas fu, resulted in:

![image](https://user-images.githubusercontent.com/3184072/113485350-50283f00-94ad-11eb-9e24-a5ae78f48488.png)

Well, it might rather be a coincidence or maybe developers had more time for some Python programming in their spare time? Who knows? What do you think?

**Comment by abravalheri** [comment]
That is kind of cool,

I wonder what would be the reason for this weird correlation...

**Comment by FlorianWilhelm** [comment]
I wonder what causes this huge increase in downloads. It seems to be only caused by the 3.* versions...

![image](https://user-images.githubusercontent.com/3184072/155733783-6eb61104-0a79-46c7-8226-0d8a680639af.png)
